src/main/java/com/phonebookservice/controller/AbstractController.java

```diff
@@ -0,0 +1,33 @@
+package main.java.com.phonebookservice.controller;
```

Comment:
Why do u have "main.java.com.phonebookservice.controller" it should be "com.phonebookservice.controller"